ldapAdapterEXT-ext-impl/src/main/resources/ldapimporter.properties

```diff
@@ -0,0 +1,24 @@
+# properties for CustomPortalLDAPImporterImpl
+
+## if the system.config.path is set, then the system-wide configuration will override settings found in this file
+#system.config.path=
+
+## when true, importer will not add user to any organizations if no org mapping has matched
+## otherwise, the organization gets searched or created as it is, without mapping
+## defaults to false
+# default.to.none=true
```

Comment:
this flag is supposed to enable mapping 1:1. Without any configuration, the code will map any LDAP organization to itself -> that's how it works before my changes. If this flag is enabled, however, all unmatched organizations will map to null. Thus, the user will have no organization.
Setting a specific default organization is a possible feature, but no one needs it at the moment.